README.md

```diff
@@ -1,6 +1,12 @@
 # Hello Microverse
```

Comment:
- [ ] Pull Request should have a summary describing introduced changes in more detail, kindly consider adding all the changes in this PR to your description

Reply:
I update the title and the description of the PR.